helm/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: zaakafhandelcomponent
 description: A Helm chart for installing Zaakafhandelcomponent
-version: 0.3.3
+version: 0.3.4
```

Comment:
What exactly is our / the best procedure for dealing with these Helm Chart versions? And should we not document that somewhere in our developer docs? Is it something like: whenever 'anything' is changed in our Helm Chart that our Helm Chart users need to know about we need to update this version? Or just any change?